elastic/kibana
# [Uptime] Show troubleshoot guide on missing monitors

## Summary

Fixes: https://github.com/elastic/uptime/issues/465

It shows the troubleshoot guide in case user has data in pings over time chart and none in monitor list. 

### Testing

1. Start heartbeat or connector remote_cluster:heartbeat using oblt cli cluster
2. Change your system clock to 5-6 mins in future
3. You will see many monitors will disappear, search for Elasticsearch monitor.
4. This will show the exact state and trouble shoot guide
